src/components/navigation/link.ts

```diff
@@ -198,6 +195,19 @@ export class Link {
             environment
           ];
     const baseUrl = `${linkProtocol}://${domain}`;
+    const isAlreadyAbsolute = link?.de.substring(0, 4) === 'http';
+    if (
+      !useAbsoluteUrls &&
```

Comment:
Not sure you care in this case for the absolute URL setting as for `(isInternal || forceAutoscoutLink || forceMotoscoutLink)` they will always become absolute / same as default case below